src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -90,13 +91,18 @@ private void SuppressedIssuesMonitor_SuppressionsUpdateRequested(object sender,
             // does all of its work on a background thread.
             try
             {
+                CodeMarkers.Instance.FileSynchronizerUpdateRequesteStart();
```

Comment:
This will tell us how often the update is requested. The actual update is happening on a different thread, and this method isn't waiting for the result so we don't know how long the update will take.

I'm not sure if it's worth recording Start/Stop if all of the work is being done on another thread, but I guess it doesn't hurt.

However, do we want to measure how long the actual update takes?